### DISCUSSION TITLE
Making a window stay at bottom

### DISCUSSION THREAD

**Discussion by xade93** [i3/i3#5298] [Q&A]
Hi all,

I am trying to find a way that makes one floating window permanently lies behind every other window (tile / floating), because that's the desired behavior of desktop sticky notes. Right now it looks like this:
![image](https://user-images.githubusercontent.com/24752033/202890871-7e07ebd1-06b4-4f7d-b3ee-e66d7dae4d2f.png)

**Comment by vincentbernat** [comment]
This is not possible with i3. Floating windows are always on top of tiling windows. Only unmanaged windows can act differently.
